tests/conftest.py

```diff
@@ -428,3 +429,44 @@ def hg_clone(hg_server, tmpdir):
     clone_dir = tmpdir.join("hg_clone")
     subprocess.run(["hg", "clone", hg_server, clone_dir.strpath], check=True)
     return clone_dir
+
+
+@pytest.fixture
+def register_codefreeze_uri(request_mocker):
+    request_mocker.register_uri(
+        "GET",
+        "https://product-details.mozilla.org/1.0/firefox_versions.json",
+        json={
+            "NEXT_SOFTFREEZE_DATE": "2122-01-01",
+            "NEXT_MERGE_DATE": "2122-01-01",
+        },
+    )
+
+
+@pytest.fixture
+def codefreeze_datetime(request_mocker):
+    today = datetime(2000, 1, 5, 0, 0, 0, tzinfo=timezone.utc)
+    freeze_date = datetime(2000, 1, 3, 0, 0, 0)
+    merge_date = datetime(2000, 1, 6, 0, 0, 0)
+    outside_freeze_date = datetime(2000, 2, 3, 0, 0, 0)
+    outside_merge_date = datetime(2000, 2, 6, 0, 0, 0)
```

Comment:
nit

```suggestion
    today = datetime(2000, 1, 5, 0, 0, 0, tzinfo=timezone.utc)
    two_days_ago = datetime(2000, 1, 3, 0, 0, 0)
    tomorrow = datetime(2000, 1, 6, 0, 0, 0)
    one_month_from_today = datetime(2000, 2, 3, 0, 0, 0)
    one_month_and_two_days_from_today = datetime(2000, 2, 6, 0, 0, 0)
```

Maybe the last two, if they don't need to be in this exact interval can be made to be `four_weeks_from_today` and `five_weeks_from_today` or something to simplify the wording a little more.